app/src/main/java/sdp/moneyrun/MenuActivity.java

```diff
@@ -13,10 +14,13 @@
 
 import com.google.android.material.navigation.NavigationView;
 
+import java.util.concurrent.TimeUnit;
+
 public class MenuActivity extends AppCompatActivity /*implements NavigationView.OnNavigationItemSelectedListener*/ {
     private Button profileButton;
 
     private Button joinGame;
+    private Button askQuestion;
```

Comment:
I suppose this will only be for testing purposes and all this logic will be moved in the appropriate stage of the game once it is implemented?

Reply:
Yes, that is correct.